base/domain-eng.yml

```diff
@@ -260,6 +260,7 @@ responses:
 
         - If you develop cough, fever, weight loss or night sweats, visit your nearest clinic.
 
+  # TODO: fix this translation
```

Comment:
This is the original english content, does it need a translation?

Reply:
probably not, thanks